app/controllers/claims/support/schools_controller.rb

```diff
@@ -8,20 +8,19 @@ def show
   end
 
   def new
-    @school = Claims::School.new
+    @school_form = SchoolForm.new
```

Comment:
I'm wondering if "school onboarding form" is a more accurate name for this. That way we leave room for other potential forms that might be needed in relation to schools – such as user input of additional fields that we can't source from GIAS.

```suggestion
    @school_onboarding_form = SchoolOnboardingForm.new
```

What do you think?

Reply:
resolved on [8a29568](https://github.com/DFE-Digital/itt-mentor-services/pull/105/commits/8a2956808a0e940e50960406e039c4df9a898787)